extensions/mongodb-client/deployment/src/test/java/io/quarkus/mongodb/NamedMongoClientConfigTest.java

```diff
@@ -0,0 +1,45 @@
+package io.quarkus.mongodb;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import javax.inject.Inject;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import com.mongodb.client.MongoClient;
+
+import io.quarkus.mongodb.runtime.MongoClientConnection;
+import io.quarkus.test.QuarkusUnitTest;
+
+public class NamedMongoClientConfigTest extends MongoWithReplicasTestBase {
```

Comment:
+1

Reply:
what do you expect here ?

Reply:
I was just indicating that this is a good change :)